R/PipeOpTorchFn.R

```diff
@@ -0,0 +1,66 @@
+#' @title Custom Function
+#' @name mlr_pipeops_nn_fn
+#' 
+#' @description
+#' Applies a user-supplied function to a tensor.
+
+#' @section Parameters:
+#' * `fn` :: `function`\cr
+#'   The function to apply. Takes a `torch` tensor as input and returns a `torch` tensor.
```

Comment:
Also document that the input tensor will be passed as the first argument to this function by position.